src/py42/clients/users.py

```diff
@@ -244,3 +244,57 @@ def change_org_assignment(self, user_id, org_id):
         uri = u"/api/UserMoveProcess"
         data = {u"userId": user_id, u"parentOrgId": org_id}
         return self._session.post(uri, data=json.dumps(data))
+
+    def get_available_roles(self):
+        """Report the list of roles that are available for the authenticated user to
+        assign to other users.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#UserMoveProcess-post>`__
```

Comment:
Probably don't want to link to UserMoveProcess api.

Reply:
woop